### PR TITLE
Completed es6-classes

### PR DESCRIPTION
**Pull request by Johnny-Baltazar**
Screenshot 2022-08-25 095046
Screenshot 2022-08-25 100804
Screenshot 2022-08-25 091807

### CODE REVIEW

File: es6-classes/stopwatch.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  seconds -= (secondsPerMinute * minutes);
+  hours = hours.toString().padStart(2, '0');
+  minutes = minutes.toString().padStart(2, '0');
+  seconds = seconds.toString().padStart(2, '0');
+  return `${hours}:${minutes}:${seconds}`;
+};
+
+Stopwatch.prototype.reset = function () {
+  this.elapsedSeconds = 0;
+};
```
Comment (scbowler) [inline es6-classes/stopwatch.js]: All the methods for `StopWatch` should be defined directly within the `StopWatch` class

File: es6-classes/student.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Student.prototype.getFullName = function () {
+  const { firstName, lastName } = this;
+  return `${firstName} ${lastName}`;
+};
+
+Student.prototype.getIntroduction = function () {
+  const { subject } = this;
+  const fullName = this.getFullName();
+  return `Hello, my name is ${fullName} and I am studying ${subject}.`;
+};
```
Comment (scbowler) [inline es6-classes/student.js]: Same here, the methods should be defined within the `Student` class.

**Review by scbowler — APPROVED** [review]
Off to a good start, but some updates needed. Let me know if you have any questions.

**Comment by Johnny-Baltazar** [comment]
Screenshot 2022-08-28 204956

**Pull request merged**